.drone.yml

```diff
@@ -16,8 +16,13 @@ pipeline:
       - go test -v -cover $(go list ./... | grep -v /vendor)
       - go build $(go list ./... | grep -v /vendor)
 
-matrix:
-  GO_VERSION:
-    - latest
-    - "1.8"
-    - "1.7"
+  test-e2e:
+    image: golang:1.8
+    environment:
+      - KETO_ASSETS_DIR=/ketoassets
+      - CLUSTER_NAME=drone-build-${DRONE_BUILD_NUMBER}
+    commands:
```

Comment:
Sounds good. Had to do this check within the scripts, because `exit 0` in drone yaml just moves to the next line in the commands section.
Flag unset: https://drone.digital.homeoffice.gov.uk/UKHomeOffice/keto/491
Flag set: https://drone.digital.homeoffice.gov.uk/UKHomeOffice/keto/492